src/kirby/directives/nativescript-only/elevation/elevation.directive.tns-only.ts

```diff
@@ -0,0 +1,53 @@
+import { Directive, OnInit, Input, ElementRef, Renderer2 } from '@angular/core';
+import { isIOS } from 'tns-core-modules/ui/page/page';
+
+import { ScssHelper } from '../../../scss/scss-helper';
+
+@Directive({ selector: '[kirbyElevation]' })
+export class NativeScriptElevationDirective implements OnInit {
+  @Input() kirbyElevation?: 2 | 4 | 8;
+  @Input() iosMasksToBounds?: boolean;
+
+  constructor(private el: ElementRef, private renderer: Renderer2) {}
+
+  ngOnInit(): void {
+    if (!this.kirbyElevation) {
+      return;
+    }
+    this.awaitNativeElementInitAndApplyElevation();
```

Comment:
Consider attaching to the `loaded` event instead - see [`VerticalTextAlignmentDirective`](https://github.com/kirbydesign/designsystem/blob/master/src/kirby/directives/vertical-text-alignment/vertical-text-alignment.directive.tns-only.ts) for an example